README.md

```diff
@@ -821,6 +827,99 @@ There are a few limitations to the APIs you should be aware of ahead of time.
 
 The file Example.psd is included in this repository if you'd like to experiment with these example calls on your own.
 
+# ImageCutout
+
+The Image Cutout API is powered by Sensei, Adobe’s Artificial Intelligence Technology, and Photoshop. The API's can identify the main subject of an image and produce two types of outputs. You can create a greyscale [mask](https://helpx.adobe.com/photoshop/using/masking-layers.html) png file that you can composite onto the original image (or any other).  You can also create a cutout where the mask has already composited onto your original image so that everything except the main subject has been removed.
```

Comment:
@noahhoffman The `mask` hyperlink takes the user to the photoshop page. That page talks about both layer mask and vector mask. Do we need to mention here, what kind of mask the user needs to create?